kernel/component-manager/src/main/java/org/sakaiproject/util/NoisierDefaultListableBeanFactory.java

```diff
@@ -42,6 +43,11 @@
 
 	public static boolean noisyClose = true;
 
+  public NoisierDefaultListableBeanFactory() {
+    // We set this particular strategy because otherwise Spring uses a non-serializable object when processing the bytecode 
+    setInstantiationStrategy(new SakaiCglibSubclassingInstantiationStrategy());
```

Comment:
This class does some AOP magic - we need it to use a specific CGLIB class so that the changes made are serializable. The class used is buried and not accessible, so a copy (renamed with "Sakai" on the front) is used.